src/clincoded/static/components/variant_central/interpretation/population.js

```diff
@@ -697,24 +872,33 @@ var CurationInterpretationPopulation = module.exports.CurationInterpretationPopu
         }
     },
 
-    // Method to render ExAC population table header content
-    renderExacHeader: function(hasExacData, loading_myVariantInfo, exac, singleNucleotide) {
-        if (hasExacData && !loading_myVariantInfo && singleNucleotide) {
-            const variantExac = exac._extra.chrom + ':' + exac._extra.pos + ' ' + exac._extra.ref + '/' + exac._extra.alt;
-            const linkoutExac = 'http:' + external_url_map['EXAC'] + exac._extra.chrom + '-' + exac._extra.pos + '-' + exac._extra.ref + '-' + exac._extra.alt;
-            return (
-                <h3 className="panel-title">ExAC {variantExac} (GRCh37)
-                    <a href="#credit-myvariant" className="credit-myvariant" title="MyVariant.info"><span>MyVariant</span></a>
-                    <a className="panel-subtitle pull-right" href={linkoutExac} target="_blank">See data in ExAC</a>
-                </h3>
-            );
-        } else {
-            return (
-                <h3 className="panel-title">ExAC
-                    <a href="#credit-myvariant" className="credit-myvariant" title="MyVariant.info"><span>MyVariant</span></a>
-                </h3>
-            );
+    // Method to render ExAC/gnomAD population table header content
+    renderExacGnomadHeader: function(datasetCheck, loading_myVariantInfo, dataset, singleNucleotide, response, datasetName) {
+        let datasetVariant = '';
+        let datasetLink;
+
+        if (datasetCheck) {
+            datasetVariant = ' ' + dataset._extra.chrom + ':' + dataset._extra.pos + ' ' + dataset._extra.ref + '/' + dataset._extra.alt + ' (GRCh37)';
+        } else if (response) {
+            let alleleData = this.parseAlleleMyVariant(response);
+
+            if (Object.keys(alleleData).length) {
+                datasetVariant = ' ' + alleleData.chrom + ':' + alleleData.pos + ' ' + alleleData.ref + '/' + alleleData.alt + ' (GRCh37)';
+            }
+        }
+
+        if (datasetCheck && !loading_myVariantInfo && singleNucleotide) {
+            const datasetVariantURLKey = (datasetName === 'ExAC') ? 'EXAC' : (datasetName === 'gnomAD') ? 'gnomAD' : null;
```

Comment:
Will this line still be needed if we directly go with `external_url_map[datasetName]`?